Create welcome-tip-sheet.md (#1246)
carpentries/carpentries.org: pages/welcome-tip-sheet.md

---
layout: page-fullwidth
title: "Welcome to The Carpentries!"
permalink: /welcome-tip-sheet/
excerpt: A guide to Carpentries activities and key links for new members
---

> We’re happy that you’re here! This document is intended to help you get started in The Carpentries community by highlighting some key links and activities. If you have questions, please get in touch with our Director of Community, Alycia Crall, by [email](mailto:alycia@carpentries.org) or [schedule a meeting](http://calendly.com/alycia-carpentries). We are here for you as you begin your journey with our organisation. Welcome!

Reviewer: ```suggestion
We are happy that you are here! This document is intended to help you get started in The Carpentries community by highlighting some key links and activities. If you have questions, please get in touch with our Director of Community, Alycia Crall, by [email](mailto:alycia@carpentries.org) or [schedule a meeting](https://calendly.com/alycia-carpentries). We are here for you as you begin your journey with our organisation. Welcome!
```

- [Email Listservs](https://carpentries.topicbox.com/latest): Browse the various options and sign up to receive relevant regional and global communications. We recommend the “Discuss” listserv, which is used for general discussion, but you can find information on [other available listservs](https://docs.carpentries.org/topic_folders/communications/tools/slack-and-email.html).
- [Newsletter](https://carpentries.org/newsletter/): Carpentry Clippings is published every two weeks and includes community news and relevant job postings.
- [Carpentries Philanthropy](https://carpentries.us14.list-manage.com/subscribe?u=46d7513c798c6bd41e5f58f4a&id=33f76196ac): Receive communications about the philanthropic work of our organisation.
- Social media: Follow our organisation’s activities on [Twitter](https://twitter.com/thecarpentries), [Facebook](https://www.facebook.com/carpentries), [LinkedIn](https://www.linkedin.com/company/the-carpentries/), and [YouTube](https://www.youtube.com/c/TheCarpentries).

### Chat with Other Community Members

> [The Carpentries Slack](http://swcarpentry.slack.com) is where our community members connect with each other, and where we share resources and opportunities to participate. If Slack is new to you, our [Slack Quickstart Guide](https://docs.google.com/document/d/1-NcL-ofnHXjJnopdWH6jLrdLHhDJeZO3/edit?usp=sharing&ouid=102968245434308327043&rtpof=true&sd=true) will help you to set up your profile and workspace, and give you an overview of how we use the platform on a day-to-day basis. All new members to Slack are automatically added to the following Slack channels:

Reviewer: could we move the Slack Quickstart to the handbook?

Author: @fmichonneau do you mean moving the content from that google doc onto the handbook, or adding a link into the handbook and linking to there?

Reviewer: I was thinking of moving the content of the google doc onto the handbook

Author: Great. I will work on that today and then add the link here.

> - #general: main channel to discuss all things Carpentries, excluding recruitment of workshop volunteers. Everyone can post here.

Reviewer: ```suggestion
- [#general](https://swcarpentry.slack.com/archives/C03LE48AY): main channel to discuss all things Carpentries, excluding recruitment of workshop volunteers. Everyone can post here.
```

> - #welcome: where new members post short bios about themselves. You’re encouraged to say hello to others here!

Reviewer: ```suggestion
- [#welcome](https://swcarpentry.slack.com/archives/CB35ZV9KL): where new members post short bios about themselves. You’re encouraged to say hello to others here!
```

> - #jobs: where you can find job opportunities relevant to our community members

Reviewer: ```suggestion
- [#jobs](https://swcarpentry.slack.com/archives/CD099JB5X): where you can find job opportunities relevant to our community members
```

> We also have over 100 optional channels, ranging from local and regional communities (e.g., #local-japan) to channels supporting specific volunteer roles (e.g., #instructors). Not sure which channels to follow? You can find additional information about our Slack Channels in [our Handbook](https://docs.carpentries.org/topic_folders/communications/tools/slack-and-email.html).

Reviewer: ```suggestion
We also have over 100 optional channels, ranging from local and regional communities (e.g., #local-japan) to channels supporting specific volunteer roles (e.g., [#instructors](https://swcarpentry.slack.com/archives/C08BVNU00)). Not sure which channels to follow? You can find additional information about our Slack Channels in [our Handbook](https://docs.carpentries.org/topic_folders/communications/tools/slack-and-email.html).
```

- Regional Community Calls: Join a call with others from your region. [Call in details](https://pad.carpentries.org/community-discussions).

### Become an Instructor
Carpentries Instructors are the core of our community. Instructors organise and teach Carpentries workshops to spread data literacy and computational skills around the world. Members of our Instructor community collaborate and share knowledge  to actively grow their instructional and technical skills. Becoming an Instructor is a great step to add inspiring people to your network and helps you to both develop and demonstrate your ability to teach effectively.

Instructor certification includes attendance at an Instructor Training event and three short “checkout” tasks. Together, these steps ensure that our Instructors are well grounded in principles of effective teaching, familiar with The Carpentries policies and practices, and are prepared to participate in and contribute to our community in the future. Learn more by visiting our [Become an Instructor](https://carpentries.org/become-instructor/) page.

> 

Reviewer 2: I'm checking in with @klbarnes20 to ask about changing this section slightly to more align with the others

Reviewer 3: I just commented in the doc that I think your changes look fine and to go ahead with them.

Reviewer 2: Thanks, @klbarnes20! @kmomar, below is the new text to include under this section.

Carpentries Instructors are the core of our community, organising and teaching workshops to spread data literacy and computational skills around the world. Becoming an Instructor is a great step to add inspiring people to your network and helps you to both develop and demonstrate your ability to teach effectively.

- [Become an Instructor](https://carpentries.org/become-instructor/): webpage with all the details

- [Lesson Maintainers](https://docs.carpentries.org/topic_folders/maintainers/maintainers.html#maintainer-guidelines): those who support the maintenance of existing lessons
- [The Carpentries Incubator](https://carpentries-incubator.org/): where community members go to collaborate on the development of new lessons.
  - [Lesson Development Study Groups](https://carpentries-incubator.github.io/study-groups/): for community members who would like to begin working on a new lesson in The Carpentries Incubator as a cohort

### Additional Resources
The Carpentries has an extensive collection of resources available to our community members. In fact, for new members, the number and diversity of resources can be overwhelming. Not to worry! Below are links to the ones you need to get you started.
- [Our Website](https://carpentries.org/): The official website of The Carpentries organisation.
- [Our Handbook](https://docs.carpentries.org/): This is your go-to resource for all things Carpentries. Not sure where to find the information you are looking for? Just type a keyword into the search box to find the information you need.

> - [Our Blog](https://carpentries.org/blog/): Collection of blog posts from staff and community members

Reviewer: ```suggestion
- [Our Blog](https://carpentries.org/blog/): Collection of blog posts from Core Team and community members
```

### Citing and reusing this tipsheet

> This tipsheet was adapted from an original by the Center for Scientific Collaboration and Community Engagement (CSCCE) under a CC BY 4.0 license doi:  10.5281/zenodo.4161782

Reviewer 2: Replace with: This tipsheet was adapted from an original by the [Center for Scientific Collaboration and Community Engagement (CSCCE)](https://www.cscce.org/) under a CC BY 4.0 license doi:  10.5281/zenodo.4161782

> This tipsheet was adapted from an original by the Center for Scientific Collaboration and Community Engagement (CSCCE) under a CC BY 4.0 license doi:  10.5281/zenodo.4161782

Reviewer: ```suggestion
This tipsheet was adapted from an original by the Center for Scientific Collaboration and Community Engagement (CSCCE) under a CC BY 4.0 license doi: [10.5281/zenodo.4161782](https://doi.org/10.5281/zenodo.4161782).
```
